<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogIn</class>
 <widget class="QDialog" name="LogIn">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log-In</string>
  </property>
  <widget class="QLabel" name="lbl_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>294</width>
     <height>445</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Img/back-modal.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_username">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>User name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_info">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 15pt;
color: #F3A71E;</string>
   </property>
   <property name="text">
    <string>First Player</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;	
	border: 1px solid white;
}

QPushButton:hover { 
	background-color: rgba(0, 187, 193, .4);
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_new">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: white;	
	border: 1px solid white;
}

QPushButton:hover { 
	background-color: rgba(0, 187, 193, .4);
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
   </property>
   <property name="text">
    <string>New user</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_login">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	color: yellow;	
	border: 1px solid yellow;
}

QPushButton:hover { 
	background-color: rgba(255, 255, 0, .2);
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_username">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: transparent;
	color: black;
}</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>251</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="title">
    <string>Password</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="btn_pat_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;	
	border-radius: 10px;
}

QPushButton:hover { 
	background-color: rgba(0, 187, 193, .4);
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Img/asteroid-1.png</normaloff>:/Img/asteroid-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pat_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;	
	border-radius: 10px;
}

QPushButton:hover { 
	background-color: rgba(0, 187, 193, .4);
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Img/asteroid-1.png</normaloff>:/Img/asteroid-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pat_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;	
	border-radius: 10px;
}

QPushButton:hover { 
	background-color: rgba(0, 187, 193, .4);
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Img/asteroid-1.png</normaloff>:/Img/asteroid-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pat_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;	
	border-radius: 10px;
}

QPushButton:hover { 
	background-color: rgba(0, 187, 193, .4);
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Img/asteroid-1.png</normaloff>:/Img/asteroid-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pat_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;	
	border-radius: 10px;
}

QPushButton:hover { 
	background-color: rgba(0, 187, 193, .4);
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Img/asteroid-1.png</normaloff>:/Img/asteroid-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pat_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: white;	
	border-radius: 10px;
}

QPushButton:hover { 
	background-color: rgba(0, 187, 193, .4);
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Img/asteroid-1.png</normaloff>:/Img/asteroid-1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	QPushButton {
	color: white;	
	border: 1px solid white;
}

QPushButton:hover { 
	background-color: rgba(243, 167, 30, .4);;
}

QPushButton:pressed { 
	color: #F3A71E;
	border: 1px solid #F3A71E;
	background-color: rgba(0, 0, 0, .4);
}</string>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>LogIn</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>400</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
